lib/DDG/Spice/Polls.pm

```diff
@@ -0,0 +1,42 @@
+package DDG::Spice::Polls;
```

Comment:
Some of these polls are not really election polls. How about `Spice::PoliticalPolls`?